Woof/WoofTests/ViewModel/DetailPetSitterViewModelTests.swift

```diff
@@ -1,6 +1,6 @@
 import XCTest
 
-final class DetailPetSitterViewModelTests: XCTestCase {
+final class DetailSitterViewModelTests: XCTestCase {
```

Comment:
The name of type doesn't match the name of file.

Reply:
Okey, fixed.